src/server/controllers/downloadController.js

```diff
@@ -1,6 +1,18 @@
+const fs = require('fs');
+
+const indexFilePath = "./src/dataBase/file-indexData.json";
+
 module.exports = function (req, res) {
-    const filePath = "src/outputData/outputData.json";
-    const fileName = "outputData.json";
+    const getFileIndex = require('../services/helpers/readFileIndex');
+
+    const indexData = fs.readFileSync(indexFilePath);
```

Comment:
still sync

Reply:
Next version it will be the most async shit you ever seen)